cocos/renderer/pipeline/RenderAdditiveLightQueue.cpp

```diff
@@ -469,5 +450,4 @@ gfx::DescriptorSet *RenderAdditiveLightQueue::getOrCreateDescriptorSet(const Lig
     return _descriptorSetMap.at(light);
 }
 
-} // namespace pipeline
-} // namespace cc
+} // namespace cc::pipeline
```

Comment:
Nested namespace rule is disabled in the config, if you changed this due to linter errors, you might need to double-check your `clang-tidy` environment setup.

Reply:
sure